ui/app/AppLayouts/Profile/Sections/Ens/TermsAndConditions.qml

```diff
@@ -315,12 +315,12 @@ Item {
         anchors.bottomMargin: Style.current.padding
         anchors.right: parent.right
         anchors.rightMargin: Style.current.padding
-        label: parseFloat(walletModel.getSNTBalance()) < 10 ? 
+        label: parseFloat(utilsModel.getSNTBalance()) < 10 ?
           //% "Not enough SNT"
           qsTrId("not-enough-snt") :
           //% "Register"
           qsTrId("ens-register")
-        disabled: parseFloat(walletModel.getSNTBalance()) < 10 || !termsAndConditionsCheckbox.checked
+        disabled: parseFloat(utilsModel.getSNTBalance()) < 10 || !termsAndConditionsCheckbox.checked
         onClicked: transactionDialog.open()
```

Comment:
Maybe while we're here, we could update this to a `StatusButton`? Wdyt?

```qml
StyledButton {
    id: startBtn
    anchors.bottom: parent.bottom
    anchors.bottomMargin: Style.current.padding
    anchors.right: parent.right
    anchors.rightMargin: Style.current.padding
    text: parseFloat(utilsModel.getSNTBalance()) < 10 ?
    //% "Not enough SNT"
    qsTrId("not-enough-snt") :
    //% "Register"
    qsTrId("ens-register")
    enabled: parseFloat(utilsModel.getSNTBalance()) >= 10 && termsAndConditionsCheckbox.checked
    onClicked: transactionDialog.open()
}
```

Reply:
hey @emizzle,  Sounds good! I also changed the back button into a StatusButton,